client/src/elements/item-preview/preview-container.html

```diff
@@ -1,11 +1,40 @@
-<template css="background-color: ${previewColor}; color: ${previewColor};">
+<template>
   <require from="./preview-container.css"></require>
   <style>
     .preview-element-container {
+      height: 100%;
       overflow-x: auto;
       overflow-y: var(--preview-container-overflow-y, auto);
-      height: 100%;
+      border: solid 1px var(--q-color-gray-4);
+    }
+
+    .preview-element-container__background {
+      background-color: var(--q-color-gray-3);
+    }
+
+    .preview-element {
+      margin: auto;
+      width: 50%;
+      min-height: calc(350px - var(--q-space-base) * 2);
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.15);
+      padding: var(--q-space-base);
+    }
+
+    .preview-element--error-box {
```

Comment:
this class should be named `preview-element__error-box` according to BEM rules as it's a child of `preview-element`. `--` is used only for modifiers.

the same applies to `preview-element--error-text`.

Reply:
Fixed in https://github.com/nzzdev/Q-editor/pull/119/commits/b2552379567e40a58634d9dd72340c3e3b1d9a17